.github/workflows/snapshot-build.yml

```diff
@@ -80,6 +80,12 @@ jobs:
             SUBSTRATE_CLI_GIT_COMMIT_HASH=${{ github.sha }}
             RUSTFLAGS=${{ matrix.platform.rustflags }}
 
+      - name: Trigger sec-scan Workflow
```

Comment:
Will block the workflow, or just trigger another one and move on?

Reply:
It shouldn't it's triggering a webhook to dispatch it to the other workflow from what I see.